### PR TITLE
add logo as default seo image

### PR DESCRIPTION
**Pull request by wgao19**
can't really test locally or during preview because it's using absolute address `https://jamstacksingapore.com/icon.png`, and it won't show up unless we build live.
should make sure nothing breaks and merge

### CODE REVIEW

**Comment by thorsten-stripe** [comment]
@wgao19 exciting, thanks. One thought, for the social media cards, would it make sense to have a rectangular icon, instead of the square icon, like the social_icon.png I just added. Wdyt?

**Comment by thchia** [comment]
Love it folks. One thing - there seems to be a slight black border, as if the icon somehow isn’t big enough, is that intentional?

**Comment by thorsten-stripe** [comment]
@thchia where do you see that? Can you provide a screenshot?

**Comment by thchia** [comment]
@thorsten-stripe yeah I was trying but I’m on iOS12 mobile right now and I think Github isn’t playing nicely 🤷‍♀️

**Comment by thchia** [comment]
It’s minor so we can address it at a later date. Maybe it’s also another artifact of iOS12.

File: src/components/seo.js [hunk 1]
```diff
@@ -66,6 +72,14 @@ function SEO({ description, lang, meta, title }) {
...
           content: metaDescription,
         },
+        {
+          name: `twitter:image`,
+          content: `${site.siteMetadata.url}${site.siteMetadata.icon}`,
+        },
+        {
+          name: `og:image`,
+          content: `${site.siteMetadata.url}${site.siteMetadata.icon}`,
+        },
```
Comment (omonk) [inline src/components/seo.js]: technically you don't actually need twitter:image if you have og:image as well as twitter fallsback

**Comment by omonk** [comment]
Looks wicked thanks @wgao19!

**Pull request merged**